bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java

```diff
@@ -495,9 +498,20 @@ public SyncObject() {
      */
     public void recoverBookieData(final BookieSocketAddress bookieSrc, final BookieSocketAddress bookieDest)
```

Comment:
I will remove this method. we don't define the binary backward compatibility for bookkeeper admin, and I doubt people depends on bookkeeper admin programmatically.

Reply:
actually I keep this method unchanged is because BookieShell is using this method. I am changing bookie shell in the subsequent change, so I leave this method unchanged.

Reply:
Good decision.